scripts/boiler-mgr.pl

```diff
@@ -31,87 +31,87 @@
 my ($BoilerName, @FileNames) = @ARGV;
 
 my $CorrectBoiler = `cat $BoilerName` or
-	die("cannot load boilerplate from $BoilerName: $!, stopped");
+    die("cannot load boilerplate from $BoilerName: $!, stopped");
 $CorrectBoiler = &trimL(&trimR($CorrectBoiler)) . "\n\n";
 
 # the first /* comment */
 my $reComment = qr{
-	/\*.*?\*/
+    /\*.*?\*/
 }xs;
 
 # Debugging section inside a boilerplate comment.
 my $reDebug = qr{
-	^[\s*]*(DEBUG:.*?)$
+    ^[\s*]*(DEBUG:.*?)$
 }mx;
 
 # Same as $reDebug, but does not match empty DEBUG: statements.
 my $reDebugFull = qr{
-	^[\s*]*(DEBUG:[^\S\n]*\S.*?)\s*$
+    ^[\s*]*(DEBUG:[^\S\n]*\S.*?)\s*$
 }mx;
 
 # Copyright-related claims inside a boilerplate comment
 my $reClaims = qr{
-	(
-		(?:
-		 AUTHOR\b(?:.|\n)*?\*[/\s]*$|	# all authors until an "empty" line
-		 ORIGINAL\s+AUTHOR\b|	# or not the latest author
-		 COPYRIGHT\b(?!\sfile)|	# or copyright (except "COPYRIGHT file")
-		 Portions\scopyright|	# or partial copyright
-		 (?<!Squid.is.Copyrighted.)\(C\)\s|	# or (C) (except "Squid is ...")
-		 Based.upon.original.+code.by\s*\n|	# or this common pearl
-		 Modified\sby\s|		# or this
-		 BASED\sON:\s			# or this
-		)		
-		.*?							# and the claim content itself
-	)$
+    (
+        (?:
+         AUTHOR\b(?:.|\n)*?\*[/\s]*$|  # all authors until an "empty" line
+         ORIGINAL\s+AUTHOR\b|          # or not the latest author
+         COPYRIGHT\b(?!\sfile)|        # or copyright (except "COPYRIGHT file")
+         Portions\scopyright|          # or partial copyright
+         (?<!Squid.is.Copyrighted.)\(C\)\s|  # or (C) (except "Squid is ...")
+         Based.upon.original.+code.by\s*\n|  # or this common pearl
+         Modified\sby\s|               # or this
+         BASED\sON:\s                  # or this
+        )
+        .*?                             # and the claim content itself
+    )$
 }xmi;
 
 # removes common claim prefixes to minimize claim noise
 my $reClaimPrefix = qr{
-	(?:ORIGINAL\s)?AUTHOR:?|
-	based\son\s|
-	based\supon\s|
-	Portions\s
+    (?:ORIGINAL\s)?AUTHOR:?|
+    based\son\s|
+    based\supon\s|
+    Portions\s
 }xi;
 
 # We have persmission to move these frequent claims to CONTRIBUTORS.
 my $reClaimsOkToMove = qr{
-	Robert.Collins|<robertc\@squid-cache.org>|<rbtcollins\@hotmail.com>|
+    Robert.Collins|<robertc\@squid-cache.org>|<rbtcollins\@hotmail.com>|
 
-	Duane.Wessels|
+    Duane.Wessels|
 
-	Francesco.Chemolli|<kinkie\@squid-cache.org>|<kinkie\@kame.usr.dsi.unimi.it>|
+    Francesco.Chemolli|<kinkie\@squid-cache.org>|
```

Comment:
This is not a whitespace change. Moreover, it is a pretty sensitive change because this code enumerates permissions to remove their (redundant) copyright statements. Please do not mix it in with whitespace/formatting changes.

We can (and probably should) discuss whether this enumeration belongs to the official repository, but this is not the right PR for that discussion.

Reply:
Reverting. I'll check in a new PR removing my by now nonexistent old email address

Reply:
> I'll check in a new PR removing my by now nonexistent old email address

Please note that these `$reClaimsOkToMove` email addresses should not be removed until they do not appear in custom master copyright claims. It does not matter whether the address is "nonexistent" _elsewhere_. Your `kame` address, for example, is still used in two claims.

If you start working on this cleanup, please consider cleaning up a few other addresses as well.